js/currencycom.js

```diff
@@ -23,36 +23,86 @@ module.exports = class currencycom extends Exchange {
             'has': {
                 'CORS': undefined,
                 'spot': true,
-                'margin': undefined, // has but not fully implemented
-                'swap': false,
+                'margin': true,
+                'swap': true,
```

Comment:
@ttodua I think this should still be false, I don't think currencycom has `swap` trading. Also, `swap` shouldn't be set to true until all the methods have swap functionality enabled, so until you can create a swap order, fetch the swap markets ... and from looking at `fetchMarkets` it looks like `swap` is always set to `false`

Reply:
currencycom has swap yes, it's not margin trading, but `LEVERAGE` (they call so) which is swap as we checked it in the past. However, I'll double check the matter per your suggestion.

Reply:
We had some discussion about it, and currencycom has swap: https://discord.com/channels/690203284119617602/921046068555313202/944277454011396136

Reply:
> We had some discussion about it, and currencycom has swap: https://discord.com/channels/690203284119617602/921046068555313202/944277454011396136

it still shouldn't be set to true though until all the methods work for swap trading

Reply:
I am not sure if that should be - if some methods are supported from swap, then we set 'true' and 'false' goes to specifically those methods that are not supported.  @kroitor please mention what is correct thing in such case.

Reply:
I mean the methods like `createOrder` and stuff, if `swap` is set to true, the `createOrder`, `fetchOrder` ... should all work for swap